src/components/visx/SparklineChart.tsx

```diff
@@ -19,6 +19,14 @@ const theme = buildChartTheme({
   gridColorDark: 'transparent',
 });
 
+// don't render the chart with 0 width and height for one frame while getting size
+function oneIfZero(arg: number) {
+  if (arg === 0) {
+    return 1;
```

Comment:
what's the behavior of just passing in 1? why not just dont render XYChart if arg is 0?

Reply:
that is a much better idea